mne/decoding/transformer.py

```diff
@@ -17,7 +18,10 @@
 
 
 class Scaler(TransformerMixin):
-    """Standardize data across channels.
+    u"""Standardize data across channels.
+
+    By default, this makes each time point (within each epoch) have
```

Comment:
> It should remove the channel mean and divide by its deviation over time samples (i.e. all time points => meaning time * epochs)

The problem I see with this approach is if there are artifactual epochs, your std dev won't be accurate. You have to use a robust measure ...